### ISSUE TITLE
no dio_query_sectcount for the atari target?

### ISSUE THREAD

**Issue by rmadugula** [cc65/cc65#2039]
Hi,

so I tried using this piece of code as per the documentation but I'd always get unresolved function call Error: Unresolved external '_dio_query_sectcount'. I just tried to find the library code in the cc65 source and it really does seem to be missing.  Unless I'm missing something?

I see sect size... but not sectcount...

**Comment by polluks** [comment]
Indeed, see [old todo list](https://wiki.cc65.org/doku.php?id=cc65:reminders#atari_library).
